src/theory/quantifiers/sygus/rcons_type_info.cpp

```diff
@@ -64,8 +64,8 @@ Node RConsTypeInfo::nextEnum()
 
 Node RConsTypeInfo::addTerm(Node n)
 {
-  std::stringstream out;
-  return d_crd->addTerm(n, false, out);
+  std::vector<Node> rewrites;
+  return d_crd->addOrGetTerm(n, rewrites);
```

Comment:
```suggestion
  return d_crd->addOrGetTerm(n, {});
```

Reply:
This is a pass by non-const reference, so this change isn't feasible.